e2e/broken-links.spec.ts

```diff
@@ -76,7 +76,7 @@ test.describe('broken-links: static server', () => {
     ({ port, stopApp } = await startApp('STATIC'));
   });
   test.afterAll(async () => {
-    await stopApp();
+    await stopApp?.();
```

Comment:
Yeah, this should be good to avoid misleading error messages. I'll fix it with types.